### DISCUSSION TITLE
Error 500 on Centos7 & 8

### DISCUSSION THREAD

**Discussion by Aryangh1379** [DirectoryLister/DirectoryLister#675] [Help & Support] [answered]
Hi, I installed DirectoryLister as explained in the site, php is installed. I tried Centos7 and also Centos8 but I get this error:
```
192.168.121.138 is currently unable to handle this request.
HTTP ERROR 500
```
![Selection_014](https://user-images.githubusercontent.com/16537843/130109816-e7af3868-a70d-4292-943e-e9db2a415dc7.png)

**Comment by PHLAK** [comment, marked as answer]
This is likely an issue with your web server configuration and not an issue with Directory Lister.

**Comment by ciscam** [comment]
@AryanGh1379 This might be the same problem: https://github.com/DirectoryLister/DirectoryLister/discussions/697#discussioncomment-1334644
Did you just clone this git? That does not work by itself.
Releases will: https://github.com/DirectoryLister/DirectoryLister/releases

**Comment by matyii** [comment]
I fixed this issue by giving read/write permission to the "cache" folder in the app folder

```chmod +777 cache```
